src/components/request/Request.tsx

```diff
@@ -72,6 +72,7 @@ const Request = () => {
               exploreDisputeTx={exploreDisputeTx}
               proposedPrice={proposedPrice}
               parsedIdentifier={parsedIdentifier}
+              request={request}
```

Comment:
we shouldnt be using this, this is provided by the query params, not the request object from the client.  query params can be multiple shapes while the request in the client will be the expected request object. 

also, to simplify further you should be just passing the data you need, which is only ancillary data, it can be exported from the `useOracleReader`

Reply:
Yeah I was having a type issue with that request object, if this is returned by the reader even better.

Reply:
Updated.